sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -24,71 +25,190 @@
 #if NET6_0_OR_GREATER
 namespace Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests
 {
-    public class DistroWebAppLiveTests : RecordedTestBase<AzureMonitorTestEnvironment>
+    public class DistroWebAppLiveTests : BaseLiveTest
     {
         private const string TestServerPort = "9998";
         private const string TestServerTarget = $"localhost:{TestServerPort}";
         private const string TestServerUrl = $"http://{TestServerTarget}/";
 
-        private LogsQueryClient? _logsQueryClient = null;
+        private const string TestServiceName = "TestName", TestServiceNamespace = "TestNamespace", TestServiceInstance = "TestInstance", TestServiceVersion = "TestVersion";
```

Comment:
```suggestion
        private const string TestServiceName = "TestServiceName", TestServiceNamespace = "TestNamespace", TestServiceInstance = "TestInstance", TestServiceVersion = "TestServiceVersion";
```

Reply:
i'll make this change now. Need to regenerate the recordings